server/src/main/java/org/candlepin/model/Content.java

```diff
@@ -148,6 +172,103 @@ public Content(String id) {
         this.setId(id);
     }
 
+    /**
+     * Creates a shallow copy of the specified source content. Owners, attributes and content are
+     * not duplicated, but the joining objects are (ContentAttribute, ContentContent, etc.).
+     * <p/>
+     * Unlike the merge method, all properties from the source content are copied, including the
+     * state of any null collections and any identifier fields.
+     *
+     * @param source
+     *  The Content instance to copy
+     */
+    protected Content(Content source) {
+        this.setUuid(source.getUuid());
+        this.setId(source.getId());
+
+        this.setCreated(source.getCreated());
```

Comment:
Since `Date` is not immutable in Java, will this create a problem where two objects are referencing the same Date object?  Should we make a defensive copy here?  `setCreated(new Date(source.getCreated().getTime()))`